tests/regression/lp-1884849/task.yaml

```diff
@@ -0,0 +1,34 @@
+summary: regression test for LP:#1884849
+details: |
+  When a mount point described by the mount profile is unmounted,
+  unmounting it again does not result in an error.
+systems: [ubuntu-16.04-64]
+prepare: |
+  # Having gimp installed, with the desktop plug connected
+  snap install gimp
```

Comment:
Could we try one of the test-snapd-* snaps or craft one ourselves instead, so that we're not affected by the changes that may no longer trigger the error scenario?

Reply:
Actually yeah, I was just lazy.